### PR TITLE
EmmaButzlerTangibleLandscapes

### PR DESCRIPTION
**Pull request by 18ebutzler1**

### CODE REVIEW

File: activities/Emma_Butzler.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=slope"]
```
Comment (petrasovaa) [inline activities/Emma_Butzler.json:5]: You are not displaying any of the layers you are computing.

File: activities/Emma_Butzler.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "scanning_params": {
+        "smooth": 10,
+        "zexag": 2,
+        "numscans": 1,
+        "interpolate": true
+      },
+      "analyses": "Emma_Butzler.py",
+      "title": "Contours, Fill, Relief, and Direction and Curvature",
+      "author": "Emma Butzler",
+      "instructions": "Change topography and observe changes in contours,curvature, Relief, Fill and Direction."
```
Comment (petrasovaa) [inline activities/Emma_Butzler.json:17]: Please fix the uppercase/lowercase and missing space.

File: activities/Emma_Butzler.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def run_relief(scanned_elev, env, **kwargs):
+    gs.run_command("r.relief", input=scanned_elev, output="relief", env=env)
+
+
+def run_fill(scanned_elev, env, **kwargs):
+    gs.run_command(
+        "r.fill.dir", input=scanned_elev, output="fill", direction="direction", env=env
+    )
```
Comment (petrasovaa) [inline activities/Emma_Butzler.py:13]: Not sure what you wanted to show here, but to actually see something useful, display the depressions, not the depressionless DEM or direction.

**Review by petrasovaa — CHANGES_REQUESTED** [review]

**Comment by 18ebutzler1** [comment]
@petrasovaa i think i addressed all your comment.

File: activities/Emma_Butzler.py (hunk 3 above)
Comment (petrasovaa) [reply activities/Emma_Butzler.py:9]: I don't think this was addressed. The output of r.fill.dir is the DEM without the depressions, but it makes sense visually to show the depression themselves. You can derive the depressions by subtracting the two DEMs.

File: Helloworld [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (petrasovaa) [inline Helloworld:1]: Please also remove this file.

**Comment by 18ebutzler1** [comment]
@petrasovaa let me know if the current .py and .json need changed

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**